### ISSUE TITLE
Render hidden - allow user to change cookie settings

### ISSUE THREAD

**Issue by crabnky** [Harborn-digital/cookie-consent-bundle#87]
Hey,

can you add option to render the form, but don't show it?
I can render if cookie consent is not set, but after this the form is not redered at all. I want to have a clickable button, so user can bring the form up and change cookie settings anytime.

**Comment by oliverfehmel** [comment]
As far as I know the form is rendered when you want it to be rendered. So if you change the check when it should be rendered or put a "hidden" element around it, you should be able to achive what you want, no?

**Comment by crabnky** [comment]
Yeah, that's one option - second is to do a ajax call rendering form when needed. Thanks. :)

**Comment by RonRademaker** [comment]
We have discontinued the development and support of this bundle. We thank you for your input.
